packages/commonwealth/server/models/community_snapshot_spaces.ts

```diff
@@ -0,0 +1,66 @@
+import * as Sequelize from 'sequelize';
+import { DataTypes } from 'sequelize';
+import { SnapshotSpaceAttributes } from './snapshot_spaces';
+import { ModelStatic, ModelInstance } from './types';
+
+export type CommunitySnapshotSpacesAttributes = {
+  id: number;
+  snapshot_space_id: string;
+  chain_id: string;
+  created_at?: Date;
+  updated_at?: Date;
+};
+
+export type CommunitySnapshotSpaceWithSpaceAttached =
+  CommunitySnapshotSpacesAttributes & {
+    snapshot_space?: SnapshotSpaceAttributes;
+  };
+
+export type CommunitySnapshotSpaceInstance =
+  ModelInstance<CommunitySnapshotSpacesAttributes>;
+
+export type CommunitySnapshotSpaceModelStatic =
+  ModelStatic<CommunitySnapshotSpaceInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes
+): CommunitySnapshotSpaceModelStatic => {
+  const CommunitySnapshotSpaces = <CommunitySnapshotSpaceModelStatic>(
+    sequelize.define(
+      'CommunitySnapshotSpaces',
+      {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true,
+        }, // autogenerated
+        snapshot_space_id: { type: Sequelize.STRING, allowNull: false },
+        chain_id: { type: Sequelize.STRING, allowNull: false },
+        created_at: { type: Sequelize.DATE, allowNull: false },
+        updated_at: { type: Sequelize.DATE, allowNull: false },
+      },
+      {
+        tableName: 'CommunitySnapshotSpaces',
+        timestamps: true,
+        underscored: true,
+        createdAt: 'created_at',
+        updatedAt: 'updated_at',
+      }
```

Comment:
How is this table going to be queried? Do we need indexes of any kind? (remember to update the migration too)

Reply:
we can add them later -- wouldn't hurt but don't want to make too many changes